File: Amphitryon/controleurs/proposerPlat/proposerPlat.php
<?php 
require_once '../../modeles/dao/Param.php';
require_once '../../modeles/dao/DBConnex.php';
require_once '../../modeles/dao/ProposerPlatDAO.php';

//$_POST["IDSERVICE"] = 1;
//$_POST["IDPLAT"] = 6;
//$_POST["DATESERVICE"] = "2024-03-08";
//$_POST["QUANTITEPROPOSEE"] = 16;
//$_POST["PRIXVENTE"] = 13.85 ;
//$_POST["QTEDISPO"] = 16;


print(json_encode(ProposerPlatDAO::proposerPlat($_POST["IDSERVICE"],$_POST["DATESERVICE"], $_POST["IDPLAT"], $_POST["QUANTITEPROPOSEE"], $_POST["PRIXVENTE"], $_POST["QTEDISPO"])));




?>

File: app/src/main/java/com/example/amphitryon/dto/ProposerPlat.java
package com.example.amphitryon.dto;

import androidx.appcompat.app.AppCompatActivity;
import java.io.Serializable;
import android.os.Bundle;

public class ProposerPlat implements Serializable{

    private int idService;
    private String dateService;
    private int idPlat;
    private int quantiteProposee;
    private double prixVente;
    private int qteDispo;
    private int qteVendue;

    public ProposerPlat(int idService, String dateService, int idPlat, int quantiteProposee, double prixVente, int qteDispo, int qteVendue){

        this.idPlat = idPlat;
        this.idService = idService;
        this.dateService = dateService;
        this.quantiteProposee = quantiteProposee;
        this.prixVente = prixVente;
        this.qteDispo = qteDispo;
        this.qteVendue = qteVendue;

    }


}
